protected/views/base/_footer.php

```diff
@@ -31,12 +31,14 @@
 		<div class="panel">
 			<ul class="inline-list">
 				<li><strong>Need help?</strong></li>
-				<?php if (Yii::app()->params['helpdesk_email']) { ?>
-					<li><?php echo Yii::app()->params['helpdesk_email'] ?></li>
-				<?php } ?>
-				<?php if (Yii::app()->params['helpdesk_phone']) { ?>
+        <?php
+        $help_email = SettingInstallation::model()->find('`key`="helpdesk_email"')['value'];
+        $help_email .= $help_email !== null ?null:Yii::app()->params['helpdesk_email']; ?>
```

Comment:
You need to fix the spacing here. Shouldn't need to tell you where.

Also, could you invert the ternary condition here? Having a null as the middle condition of a ternary doesn't seem like a good standard to set. Alternatively, use the coalesce operator ?:

Reply:
Coalesce doesn't work in this case, it puts a 1 in. I'll invert the ternary.